common/ferrostar/src/simulation.rs

```diff
@@ -2,6 +2,11 @@ use crate::algorithms::trunc_float;
 use crate::models::{CourseOverGround, GeographicCoordinate, Route, UserLocation};
 use geo::{coord, DensifyHaversine, GeodesicBearing, LineString, Point};
 use polyline::decode_polyline;
+use serde::{Deserialize, Serialize};
```

Comment:
I think this _probably_ needs to be feature gated (unless I missed a reason we need it for other targets).